packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 42.2.0 - February 5, 2018
+### Added
+#### salesforcedx-vscode-core
+
+* Code action to run apex tests; run a single method or all test methods in a test class ([PR #291](https://github.com/forcedotcom/salesforcedx-vscode/pull/291))
+* Command to bootstrap a project for ISV debugging ([PR #282](https://github.com/forcedotcom/salesforcedx-vscode/pull/282))
```

Comment:
Let's not mention this since it currently doesn't do anything useful. It doesn't actually fetch the managed packages yet.